package.json

```diff
@@ -75,6 +75,21 @@
                     "type": "string",
                     "default": "^8.47.0",
                     "description": "%extension.commands.salesforce-mobile-offline.eslint.version%"
+                },
+                "salesforce.salesforcedx-vscode-mobile.diagnostics.suppressAll": {
```

Comment:
add 3 diagnostic setting, so developer could manually change it from vscode setting ui.

Reply:
the are visible to developer like below, will need an official wording for them:  the id and description.
![image](https://github.com/user-attachments/assets/8a4419fa-fe53-4fae-9e1d-088ec8e7358d)

Reply:
the max number of problems are removed, team deem it's not required.